pandas/_libs/tslibs/src/datetime/np_datetime_strings.c

```diff
@@ -900,8 +1271,8 @@ int make_iso_8601_datetime(npy_datetimestruct *dts, char *outstr, int outlen,
 
 string_too_short:
     PyErr_Format(PyExc_RuntimeError,
-                 "The string provided for NumPy ISO datetime formatting "
-                 "was too short, with length %d",
-                 outlen);
+                "The string provided for NumPy ISO datetime formatting "
+                "was too short, with length %"NPY_INTP_FMT,
```

Comment:
should there be a space or percent or something after the close-quote?

Reply:
the leading spaces you deleted here, were those to make a linter happy?  if not, they look wierd to me

Reply:
This would have been a straight copy / paste from NumPy. I did use clang-format for objToJSON but not the vendored items